packages/scandipwa/src/component/MyAccountDownloadable/MyAccountDownloadable.container.js

```diff
@@ -13,6 +13,7 @@ import PropTypes from 'prop-types';
 import { PureComponent } from 'react';
 import { connect } from 'react-redux';
 
+import { OrderDispatcher } from 'Component/MyAccountMyOrders/MyAccountMyOrders.container';
```

Comment:
Use async imports to import dispatchers, please. The current solution implies that in 100% cases these two files will be in the same bundle, which may not be the case.